deps/publishing/src/logseq/publishing/html.cljs

```diff
@@ -121,6 +121,7 @@ necessary db filtering"
         }
       }(window.location))"]
             ;; TODO: should make this configurable
+            [:script {:src "static/js/custom.js"}]
```

Comment:
This leads to a missing file if custom.js is not configured.

Reply:
How can it be modified so that it's only added if there's a custom.js file?

For instance, export.css is added the same way in this file, but it's not automatically added to new Logseq graphs. It has to be manually added by the user (if needed), the same way that custom.js is added.

Reply:
@andelf This should be ok because the change ensures that custom.js is always written, in the same way that custom.css is